CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Enhancements
 * UUID allowed as partition value ([#4500](https://github.com/realm/realm-core/issues/4500))
 * The error message when the intial steps of opening a Realm file fails is now more descriptive.
+* Make conversion of Decimal128 to/from string work for whole range.
```

Comment:
```suggestion
* Make conversion of `Decimal128` to/from string work for very large numbers, too. ([#4548](https://github.com/realm/realm-core/issues/4548))
```

Reply:
Technically, it's very large or very precise. E.g. `1.123456789098765432123456789` is not particularly large, but would fail to parse/serialize previously.

Reply:
Alright, very large or very precise sounds good too.